Dockerfile

```diff
@@ -158,7 +160,7 @@ COPY ./docker/sshd_entrypoint.sh /sshd_entrypoint.sh
 RUN chmod 744 /sshd_entrypoint.sh
 
 RUN mkdir /guidelines && cd /guidelines \
-  && wget https://raw.githubusercontent.com/aica-technology/.github/v0.9.0/guidelines/.clang-format
+  && wget https://raw.githubusercontent.com/aica-technology/.github/v0.9.0/guidelines/.clang-format # TODO: consider update
```

Comment:
do we want to go 1.0.4, or should we wait until we format the repo before changing the Dockerfile? `.clang-format` is technically the same

Reply:
Is there a diff?

Reply:
they are identical, so it's really just the tag (but since I'm modifying the Dockerfile we might as well...)

Reply:
As you prefer